src/components/TeachersPage/TeachersItem/TeachersItem.tsx

```diff
@@ -0,0 +1,143 @@
+import React, { useState } from 'react'
+
+import { ButtonSize, ButtonType } from '../../../helpers/types/types.ts'
+import { CustomModal } from '../../../UI-components'
+import { CustomButton } from '../../../UI-components/CustomButton/CustomButton.tsx'
+
+import styles from './TeachersItem.module.scss'
+import TeacherPopUp from '../TeacherPopUp/TeacherPopUp.tsx'
+
+interface Teacher {
+  avatar_url: string
+  lessons_done: number
+  rating: number
+  price_per_hour: number
+  name: string
+  surname: string
+  languages: string[]
+  lesson_info: string
+  conditions: string[]
+  experience: string
+  reviews?: Array<{
+    reviewer_name: string
+    reviewer_rating: number
+    comment: string
+  }>
+}
+
+interface TeacherItemProps {
+  teacher: Teacher
+}
+
+const TeacherItem: React.FC<TeacherItemProps> = ({ teacher }) => {
+  const [isExpanded, setIsExpanded] = useState(false)
+  const [isModalOpen, setIsModalOpen] = useState(false)
+
+  const toggleCard = () => {
+    setIsExpanded((prev) => !prev)
+  }
+
+  const openModal = () => {
+    setIsModalOpen(true)
+  }
+
+  return (
+    <li className={styles.teacherContainer}>
+      <div className={styles.teacherImageWrapper}>
+        <img
+          className={styles.teacherImage}
+          src={teacher.avatar_url}
+          alt={`${teacher.name} avatar`}
+          width="120"
+          height="120"
+        />
+      </div>
+      <div className={styles.teacherContent}>
+        <div className={styles.teacherTopInfo}>
+          <p className={styles.teacherLanguages}>Languages</p>
+          <p className={styles.teacherDesc}>Lessons online</p>
+          <p className={styles.teacherDesc}>
+            Lessons done: {teacher.lessons_done}
+          </p>
+          <p className={styles.teacherDesc}>Rating: {teacher.rating}</p>
+          <p className={styles.teacherDesc}>
+            Price / 1 hour: {teacher.price_per_hour}
+          </p>
+        </div>
+
+        <h3 className={styles.teacherName}>
+          {teacher.name} {teacher.surname}
+        </h3>
+        <p className={styles.teacherSpeaks}>
+          Speaks:{' '}
```

Comment:
```suggestion
          Speaks:
```
remove it , if u dont need it